bitnami/discourse/templates/deployment-sidekiq.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "discourse.fullname" . }}-sidekiq
+  labels:
+    {{- include "discourse.labels" . | nindent 4 }}
+    component: sidekiq
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "discourse.selectorLabels" . | nindent 6 }}
+      component: sidekiq
+  template:
+    metadata:
+      labels:
+        {{- include "discourse.selectorLabels" . | nindent 8 }}
+        component: sidekiq
+    spec:
+    {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      serviceAccountName: {{ include "discourse.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default "latest" }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command: ["/app-entrypoint.sh"]
+          args:
+            - nami
+            - start
+            - --foreground
+            - discourse-sidekiq
+          env:
+            - name: DISCOURSE_HOST
+              value: {{ include "discourse.fullname" . }}
+            - name: DISCOURSE_PORT
+              value: "3000"
+          envFrom:
+            - configMapRef:
+                name: {{ include "discourse.fullname" . }}-envs
+            - secretRef:
+                name: {{ include "discourse.fullname" . }}-envs
+            # {{ .Values.additionalEnvs | toYaml | indent 6 }}
+          # livenessProbe:
+          #   httpGet:
+          #     path: /
+          #     port: http
+          readinessProbe:
+            exec:
+              command: [nami status discourse-sidekiqa]
```

Comment:
I think that the httpGet liveness and readinessProbe fits better. Also, add the variables for setting the different probe settings (already suggested in the yaml)

Reply:
I don't believe the sidekiq had a http endpoint to `httpGet` against.

Reply:
Sorry, mistook it with the default discourse

Reply:
done